src/index.js

```diff
@@ -17,5 +17,36 @@ document.addEventListener('keyup', (event) => {
   }
 });
 
+// complete always is false if refresh the page
+toDoList.list.forEach((todo) => {
+  todo.complete = false;
+  localStorage.setItem('list', JSON.stringify(toDoList.list));
+});
+
+// checkbox true-false
+document.addEventListener('change', (e) => {
+  if (e.target.classList.contains('checkbox')) {
+    const { index } = e.target.dataset;
+    const db = JSON.parse(localStorage.getItem('list'));
+    toDoList.list[index].complete = !db[index].complete;
+    localStorage.setItem('list', JSON.stringify(toDoList.list));
+    const taskDescription = document.querySelectorAll('.taskElement');
+    taskDescription[index].classList.toggle('line-through');
+  }
+});
+
+// clear all button
+const clearBtn = document.querySelector('.clear');
+
+clearBtn.addEventListener('click', () => {
+  const taskDescription = document.querySelectorAll('.taskElement');
+  taskDescription.forEach((element) => {
+    if (element.classList.contains('line-through')) {
+      const index = element.dataset.taskIndex;
+      toDoList.remove(index);
+    }
+  });
+});
```

Comment:
- [ ] Please I highly recommend using the `.filter()` method in your `clear` class for the "Clear all completed" functionality per the project requirements.

![Screenshot (279)](https://user-images.githubusercontent.com/78104544/236357657-8fe5fa68-247f-400d-a741-b4d1dfdb29df.png)